alot/commands/thread.py

```diff
@@ -221,15 +221,27 @@ def apply(self, ui):
                 if sender_address not in my_addresses:
                     cleared = self.clear_my_address(
                         my_addresses, mail.get_all('To', []))
-                    recipients.append(cleared)
+                    recipients.extend(cleared)
 
                 # copy cc for group-replies
                 if 'Cc' in mail:
                     cc = self.clear_my_address(
                         my_addresses, mail.get_all('Cc', []))
-                    envelope.add('Cc', decode_header(cc))
+                    envelope.add('Cc', decode_header(', '.join(cc)))
+
+        def make_unique(value):
+            res = dict()
+            for name, address in getaddresses(value):
+                res[address] = name
+            new_value = []
+            for address, name in res.items():
+                if name != '':
+                    new_value.append('"%s" <%s>' % (name, address))
+                else:
+                    new_value.append(address)
```

Comment:
What about using a comprehension?
```python
sorted(['"{}" <{}>'.format(n, a) if n != '' else a for n, a in res.iteritems()])
```
Or do you think having the ternary in the comprehension is too complex?